### PR TITLE
DIP-205 Fixed Invalid attribute causing stacktrace in logs

### PR DESCRIPTION
**Pull request by cameron-white-janrain**

### CODE REVIEW

File: bluekai/job.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def run(job, writter, config, logger, datalib, converter):
     except Exception as ex:
 
         err = str(ex)
-        logger.exception(ex)
+        logger.error(err)
```
Comment (ebeesonjanrain) [inline bluekai/job.py]: you might want to log the exception stack trace if DEBUG is enabled

Comment (cameron-white-janrain) [inline bluekai/job.py]: Done!

**Comment by ebeesonjanrain** [comment]
:+1:

**Comment by ScottEwing** [comment]
:shipit:

**Pull request merged**